language-plutus-core/test/Evaluation/ApplyBuiltinName.hs

```diff
@@ -0,0 +1,176 @@
+-- | Constant application tests.
+
+{-# LANGUAGE GADTs      #-}
+{-# LANGUAGE RankNTypes #-}
+
+module Evaluation.ApplyBuiltinName
+    ( test_applyBuiltinName
+    ) where
```

Comment:
Merged a few files together, because after we'd deleted sizes it became pointless to keep them separate, because the `Failure` and `SuccessFailure` files were deleted and now we only have success tests.